Source/WebKit/Scripts/compile-sandbox.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+
+SANDBOX_PATH=$1;
+SANDBOX_NAME=$2;
+SDK_NAME=$3
+SANDBOX_IMPORT_DIR=$4
+
+xcrun --sdk $SDK_NAME -f sbutil;
```

Comment:
Before you land, let's silence the error from xcrun here.
```suggestion
xcrun --sdk $SDK_NAME -f sbutil 2>/dev/null;
```

Reply:
Oh, actually, we can't use xcrun like this at all — `set -e` means that a failure here aborts the script, which is presumably why it's failing on some builders.

How about testing it like this?
```
if xcrun --sdk $SDK_NAME -f sbutil 2> /dev/null; then
   # ...
fi
```

Reply:
Ah, good point! I have fixed this in the latest patch.

Thanks for reviewing!